.github/actions/check_test_project_fixture/check_test_project_fixture.mjs

```diff
@@ -0,0 +1,46 @@
+import { getInput } from '@actions/core'
+
+// If the PR has the "fixture-ok" label, just pass.
+const { labels } = JSON.parse(getInput('labels'))
+const hasFixtureOkLabel = labels.some(label => label.name === 'fixture-ok')
+
+if (hasFixtureOkLabel) {
+  console.log('Skipping check because of the "fixture-ok" label')
+} else {
+  // Check if the PR rebuilds the fixture. If it does, that's enough.
+  const { exec, getExecOutput } = await import('@actions/exec')
+  await exec('git fetch origin main')
+
+  const { stdout } = await getExecOutput('git diff origin/main --name-only')
+  const changedFiles = stdout.toString().trim().split('\n').filter(Boolean)
+  const didRebuildFixture = changedFiles.some(file => file.startsWith('__fixtures__/test-project'))
+
+  if (didRebuildFixture) {
+    console.log([
+      '',
+      "The fixture's been rebuilt"
+    ].join('\n'))
+  } else {
+    // If it doesn't, does it need to be rebuilt? If not, no problem. Otherwise, throw.
+    const shouldRebuildFixture = changedFiles.some(file => file.startsWith('packages/cli/src/commands/generate') ||
+      file.startsWith('packages/cli/src/commands/setup'))
```

Comment:
@jtoar - just realised, we should also check for changes in `packages/create-redwood-app/template`

Reply:
Sounds good I'll add that!

Reply:
Added in https://github.com/redwoodjs/redwood/pull/6072/commits/53d621de3aeb99a7b29d07d6ab738af1ee1658e0